src/user_options.c

```diff
@@ -63,6 +63,7 @@ static const struct option long_options[] =
   {"hex-salt",                  no_argument,       NULL, IDX_HEX_SALT},
   {"hex-wordlist",              no_argument,       NULL, IDX_HEX_WORDLIST},
   {"hook-threads",              required_argument, NULL, IDX_HOOK_THREADS},
+  {"identify",                  no_argument,       NULL, IDX_AUTODETECT_ONLY},
```

Comment:
please also keep the macro synchrouns with the variable name. so rename the identify to autodetect_only (or vice versa).